commcare_connect/opportunity/views.py

```
@@ -931,6 +931,13 @@ def reject_visits(request, org_slug=None, opp_id=None):
 @org_member_required
 def fetch_attachment(self, org_slug, blob_id):
```

Ruff (0.14.4)

Unused function argument: `org_slug`

(ARG001)